cloud/services/loadbalancers.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 const (
-	defaultApiserverLBPort    = 6443
+	DefaultApiserverLBPort    = 6443
 	defaultKonnectivityLBPort = 8132
```

Comment:
worth it just making both of these public and configurable while we're here?

Reply:
Sounds good to me